pkg/health_server/health_server.go

```diff
@@ -24,12 +24,11 @@ import (
 	"github.com/ipdk-io/k8s-infra-offload/pkg/types"
 	"github.com/ipdk-io/k8s-infra-offload/pkg/utils"
 	"github.com/sirupsen/logrus"
-	"google.golang.org/grpc"
 	"gopkg.in/tomb.v2"
 )
 
 var (
-	grpcDial       = grpc.Dial
+	grpcDial       = utils.GrpcDialInsecure
```

Comment:
Is it always insecure now?

Reply:
This function is only used for health check of cni server. For agent-manager communication, we dial in with credentials (utils.GrpcDialWithCred)